site/content/api/Credentials-API.md

```diff
@@ -3,7 +3,7 @@ title = "Credentials API"
 weight = 430
 +++
 
-The *Credentials API* is used by *Protocol Adapters* to retrieve credentials used to authenticate *Devices* connecting to the adapter. In particular, the API supports the storage, look up and deletion of *shared secrets* which are often used by IoT devices by means of *username/password* based authentication schemes.
+Eclipse Hono&trade;'s *Credentials API* is used by *Protocol Adapters* to retrieve credentials used to authenticate *Devices* connecting to the adapter. In particular, the API supports the storage, look up and deletion of *shared secrets* which are often used by IoT devices by means of *username/password* based authentication schemes.
```

Comment:
Although it is not wrong, simply using Hono is sufficient. I also would prefer Eclipse Hono. But repeating the ™ is not necessary, as the document will be rendered by the Hono theme, which already shows that at the first and most prominent place. Also see: https://www.eclipse.org/legal/logo_guidelines.php#Proper_Notice_and_Attribution

Reply:
I see your point. We are currently using `Eclipse Hono&trade;` on every page when we first mention Hono. FMPOV we should instead modify the theme to render Eclipse Hono with the TM symbol in the orange header of the page and then simply refer to Hono as just *Hono* on all content pages. WDYT?

Reply:
I will do this in a separate commit/PR.